<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <group ns="tj2" >
        <node pkg="tj2_pursuit" type="tj2_pursuit_node.py" respawn="false" name="tj2_pursuit" output="screen">
            <remap from="detections" to="cargo/detections"/>
            <param name="class_names_path" value="$(find tj2_yolo)/models/cargo_2022.names"/>

            <!-- <remap from="detections" to="powercell/detections"/> -->
            <!-- <param name="class_names_path" value="$(find tj2_yolo)/models/powercell_2021.names"/> -->
            <remap from="cmd_vel" to="cmd_vel_follow"/>
            <param name="distance_offset" value="-0.0"/>
            <param name="distance_threshold" value="0.05"/>
            <param name="send_rate" value="3.0"/>
            <param name="angular_velocity" value="1.75"/>
            <param name="direction_change_interval" value="1.5"/>
        </node>
    </group>
</launch>
